src/containers/Onboarding/CreateGroup.js

```diff
@@ -40,7 +43,37 @@ class CreateGroup extends Component {
   * @param {Object} fields
   */
   _handleSubmit(fields) {
-    return this.props.createGroup(fields);
+    const promise = new Promise((resolve) => {
+      resolve(this.props.createGroup(fields));
+    });
+
+    const index = {
+      confirmation: 0,
+      delivery: 1
+    };
+
+    promise.then((group) => {
+      const rule = new RRule({
```

Comment:
I need your help here. When I wrap this into a try catch, I get `TypeError: _rrule.RRule is not a constructor`... And the `import` seems to be correct.